onvm/onvm_mgr/onvm_stats.h

```diff
@@ -63,6 +63,15 @@
 #define ONVM_JSON_EVENTS_FILE ONVM_STATS_PATH_BASE "onvm_json_events.json"
 #define ONVM_STATS_FILE ONVM_STATS_PATH_BASE "onvm_stats.txt"
 
+/*
+ * Handle types of web stats events
```

Comment:
A bit overkill, for short comments (not method stub comments) 
`/* Handle types of web stats events */`